online-shop/src/components/product/product.styles.tsx

```diff
@@ -0,0 +1,69 @@
+import {makeStyles} from "@material-ui/core";
```

Comment:
I highly advise you to install and configure prettier for this project. I see a lot of bad formatting (such as 3 newlines one after another without any reason, especially within an object which is bad practice); prettier should fix it automatically for you every time you call it.